lib/release/notes/log.rb

```diff
@@ -67,7 +67,10 @@ def copy_single_tag_of_activity(tag_from:, tag_to: "HEAD")
       # @api private
       def find_last_tag_and_log
         last_tag = system_last_tag.delete!("\n")
```

Comment:
If rubocop yells at you - you can prob move this to it's own method after rebasing with master:

```ruby
def last_tag
  system_last_tag.strip
end
```

That should reduce complexity.